### PR TITLE
update to 1.1.0

### PR DESCRIPTION
**Pull request by vstam1**
- [x] Upgrade dependencies to 1.1.0
- [x] Fix tests
- [x] Fix benchmarks
- [x] Fix migrations (pass try-runtime)
- [x] Update spec version
- [x] Run benchmarks (update weights)

### CODE REVIEW

File: runtime/integration-tests/src/xcm_transfers.rs [hunk 1]
```diff
@@ -202,52 +204,54 @@ fn transfer_ksm_from_sibling() {
...
+            WeightLimit::Unlimited
         ));
     });
 
-    #[cfg(not(feature = "runtime-benchmarks"))]
-    Development::execute_with(|| {
-        let ksm_balance = OrmlTokens::free_balance(CurrencyId::KSM, &SiblingKusamaReceiver::get());
-        assert!(ksm_balance > 0);
-    });
+    // TODO: Fix this test
```
Comment (f-gate) [inline runtime/integration-tests/src/xcm_transfers.rs:217]: whats up with this?

**Review by f-gate — APPROVED** [review]
lgtm

File: runtime/integration-tests/src/xcm_transfers.rs (hunk 1 above)
Comment (samelamin) [reply runtime/integration-tests/src/xcm_transfers.rs]: @vstam1 out of curiosity why is the test failing?

Comment (vstam1) [reply runtime/integration-tests/src/xcm_transfers.rs]: I fixed the test now. Looked like the relay chain wasn't processing the messages if `Kusama::execute_with` wasn't called.

**Comment by samelamin** [comment]
LGTM once build is green! Thanks @vstam1!

**Review by samelamin — APPROVED** [review]

**Comment by samelamin** [comment]
looks like there is a failing test on crowdfunding 

https://github.com/ImbueNetwork/imbue/actions/runs/6789381511/job/18456495904?pr=268

but im merging since crowdfunding isnt used

**Pull request merged**